src/main/java/org/ihtsdo/snowowl/authoring/single/api/service/UiStateService.java

```diff
@@ -3,78 +3,104 @@
+			return resourceService.read(getTaskUserPanelPath(projectKey, taskKey, username, panelId));
 		} catch (NoSuchFileException e) {
 			throw new ResourceNotFoundException("ui-state", panelId);
 		}
 	}
 
-	public String retrieveTaskPanelStateWithoutThrowingResourceNotFoundException(String projectKey, String taskKey, String username, String panelId) throws IOException {
+	public String retrieveTaskPanelStateWithoutThrowingResourceNotFoundException(final String projectKey,
+																				 final String taskKey,
+																				 final String username,
+																				 final String panelId)
+			throws IOException {
 		try {
-			return arbitraryJsonService.read(getTaskUserPanelPath(projectKey, taskKey, username, panelId));
+			return resourceService.read(getTaskUserPanelPath(projectKey, taskKey, username, panelId));
 		} catch (NoSuchFileException e) {
 			return null;
 		}
 	}
 
-	public void persistPanelState(String username, String panelId, String jsonState) throws IOException {
-		arbitraryJsonService.write(getUserPanelPath(username, panelId), jsonState);
+	public void persistPanelState(final String username,
+								  final String panelId,
+								  final String jsonState) throws IOException {
+		resourceService.write(getUserPanelPath(username, panelId), jsonState);
 	}
 
-	public String retrievePanelState(String username, String panelId) throws IOException {
+	public String retrievePanelState(final String username,
+									 final String panelId) throws IOException {
 		try {
-			return arbitraryJsonService.read(getUserPanelPath(username, panelId));
+			return resourceService.read(getUserPanelPath(username, panelId));
 		} catch (NoSuchFileException e) {
 			throw new ResourceNotFoundException("ui-state", panelId);
 		}
 	}
 	
-	private String getTaskUserPath(String projectKey, String taskKey, String username) {
+	private String getTaskUserPath(final String projectKey,
+								   final String taskKey,
+								   final String username) {
 		return projectKey + "/" + taskKey + "/user/" + username + "/ui-panel/";
 	}
 
-	private String getTaskUserPanelPath(String projectKey, String taskKey, String username, String panelId) {
+	private String getTaskUserPanelPath(final String projectKey,
+										final String taskKey,
+										final String username,
+										final String panelId) {
 		return getTaskUserPath(projectKey, taskKey, username) + panelId + ".json";
 	}
 
-	private String getUserPanelPath(String username, String panelId) {
+	private String getUserPanelPath(final String username,
+									final String panelId) {
 		return "/user/" + username + "/ui-panel/" + panelId + ".json";
 	}
 
 
-	public void deleteTaskPanelState(String projectKey, String taskKey, String username, String panelId) {
-		arbitraryJsonService.delete(getTaskUserPanelPath(projectKey, taskKey, username, panelId));
+	public void deleteTaskPanelState(final String projectKey,
```

Comment:
I'm not a fan of this code style. It only looks good if everyone has the same IDE tab size. It looks pretty bad in github.
<img width="994" alt="Screenshot 2020-11-26 at 09 37 49" src="https://user-images.githubusercontent.com/2010517/100334169-54a68d80-2fcb-11eb-8536-565c2216fc8c.png">

I don't want to dictate tight code style but will share what Michael and I do in case you get on with it:
Parameters on a single line unless they go over 180 characters. If they do then two lines is fine. Indent the second line just the default amount rather than trying to get them to align with the first parameters as it doesn't seem to work consistently and the indentation can be really big with long method names. This is more consistently readable. When using two lines for params I tend to put a blank line at the start of the method body to give good separation between the params and body. There are exceptions to this, for example in rest controllers where the params can have three annotations each (if we write swagger notes) then we just stick them all on their own line as there is lots to read there. Still we just indent the default amount. I hope that helps you understand what we have at the moment in most places (probably excluding reporting).

Reply:
Thanks, noted for the future. I've made the changes to the styling.